src/Our.Umbraco.DocTypeGridEditor/Web/UI/App_Plugins/DocTypeGridEditor/Js/doctypegrideditor.resources.js

```diff
@@ -2,14 +2,14 @@
     function ($q, $http, umbRequestHelper) {
         return {
             getContentTypeAliasByGuid: function (guid) {
-                var url = "/umbraco/backoffice/DocTypeGridEditorApi/DocTypeGridEditorApi/GetContentTypeAliasByGuid?guid=" + guid;
+            	var url = umbRequestHelper.convertVirtualToAbsolutePath("~/umbraco/backoffice/DocTypeGridEditorApi/DocTypeGridEditorApi/GetContentTypeAliasByGuid?guid=" + guid);
```

Comment:
@0Neji Sorry to sound picky, any chance you can correct the whitespacing - tab to spaces? Thanks.